docs/install-machine.md

```diff
@@ -57,6 +57,21 @@ to your PATH.
         $ docker-machine -v
         machine version 0.5.0 (3e06852)
 
+## Installing bash completion scripts
+
+There are a [number of scripts](../contrib/completion/bash) which add command
```

Comment:
I'm not sure if this link is correct - @moxiegirl can you comment?  It's meant to link to source code in the docker machine repo.